client/my-sites/exporter/guided-transfer-options.jsx

```diff
@@ -18,7 +19,13 @@ export default React.createClass( {
 	},
 
 	purchaseGuidedTransfer() {
-		page( `/settings/export/${this.props.siteSlug}/guided` );
+		const { siteSlug } = this.props;
+		if ( config.isEnabled( 'manage/export/guided-transfer' ) ) {
+			page( `/settings/export/${ siteSlug }/guided` );
+		} else {
+			// Redirect to legacy guided transfer
+			window.location = `https://${ siteSlug }/wp-admin/paid-upgrades.php?product=40&view=purchase&source=export`;
```

Comment:
> I think we should use the site url here instead of siteSlug

Ah, good point. I'll update.

> Also I'm not sure about the product=40 part.

Yeah that does feel a bit icky, but once #6143 and #6259 lands we can fetch the product id (and price) from the store.